tests/test_pyliquibase.py

```diff
@@ -50,8 +50,9 @@ def test_exception(self):
             lb.status()
         except Exception as e:
             self.assertTrue("Liquibase execution failed" in str(e))
-
+    
     def test_download_additional_java_library(self):
         lb = Pyliquibase(defaultsFile=os.path.dirname(os.path.realpath(__file__)) + "/resources/liquibase.properties")
-        lb.download_additional_java_library(url="https://github.com/liquibase/liquibase-snowflake/releases/download/liquibase-snowflake-4.11.0/liquibase-snowflake-4.11.0.jar")
-        lb.download_additional_java_library(url="https://github.com/liquibase/liquibase-snowflake/releases/download/liquibase-snowflake-4.11.0/liquibase-snowflake-4.11.0.jar")
+        lb.download_additional_java_library(url="https://github.com/liquibase/liquibase-snowflake/releases/download/liquibase-bigquery-4.21.1/liquibase-bigquery-4.21.1.jar")
```

Comment:
what are the changes here about? seems like unintended changes ? if yes could we rollback this changes?

Reply:
The snowflake extension is deprecated, so I changed the test to another extension still used by the project

Reply:
this part is more relevant to testing `download_additional_java_library` method we can leave it as before.

Reply:
Even if snowflake extension was removed from the project?

Reply:
I made a mistake. I forgot to change the first part of the link

https://github.com/liquibase/liquibase-snowflake/releases/download/liquibase-bigquery-4.21.1/liquibase-bigquery-4.21.1.jar

should be

https://github.com/liquibase/liquibase-bigquery/releases/download/liquibase-bigquery-4.21.1/liquibase-bigquery-4.21.1.jar